app/src/main/java/be/chvp/nanoledger/ui/common/TransactionFormViewModel.kt

```diff
@@ -150,20 +157,29 @@ abstract class TransactionFormViewModel
             transaction.append('\n')
             // Drop last element, it should always be an empty posting (and the only empty posting)
             for (posting in postings.value!!.dropLast(1)) {
-                val usedLength = 7 + posting.first.length + posting.second.length + posting.third.length
+                val usedLength =
+                    7 +
+                        posting.first.length +
+                        posting.second.length +
+                        posting.third.length
+
```

Comment:
Was this change done by the linter? I personally find the one on a single line clearer.

Reply:
This one is because I added the posting.fourth.lenght, but removed later because it has no sense to count the note length in the spaces count. I'll restore this